nodes/index.adoc

```diff
@@ -129,3 +129,53 @@ As an administrator, You can perform various tasks on a Linux container, such as
 {product-title} provides specialized containers called xref:../nodes/containers/nodes-containers-init.adoc#nodes-containers-init[Init containers]. Init containers run before application containers and can contain utilities or setup scripts not present in an application image. You can use an Init container to perform tasks before the rest of a pod is deployed.
 
 Apart from performing specific tasks on nodes, pods, and containers, you can work with the overall {product-title} cluster to keep the cluster efficient and the application pods highly available.
+
+[id="commonterms-node"]
+== Common terms
+
+This glossary defines common terms that are used in the _node_ content. These terms help you understand _nodes_ effectively.
+
+Container::
```

Comment:
It might be helpful to add discrete headings to these glossary entries like:

```suggestion
[discrete]
[id="commonterms-node-container"]
=== Container

*the definition*
```
Just a suggestion. :)